main.tf

```diff
@@ -1 +1,188 @@
-# Your Terraform code goes here :)
+locals {
+  org_integration = var.integration_type == "ORGANIZATION"
+  project_id = data.google_project.selected.project_id
+  sink_name = length(var.existing_sink_name) > 0 ? var.existing_sink_name : (
+    local.org_integration ? "${var.prefix}-${var.organization_id}-lacework-sink-${random_id.uniq.hex}" : "${var.prefix}-lacework-sink-${random_id.uniq.hex}"
+  )
+  logging_sink_writer_identity = length(var.existing_sink_name) > 0 ? null : (
+    (local.org_integration) ? (
+      [google_logging_organization_sink.lacework_organization_sink[0].writer_identity]
+      ) : (
+      [google_logging_project_sink.lacework_project_sink[0].writer_identity]
+    )
+  )
+  service_account_name = var.use_existing_service_account ? (
+    var.service_account_name
+    ) : (
+    length(var.service_account_name) > 0 ? var.service_account_name : "${var.prefix}-${random_id.uniq.hex}"
+  )
+  service_account_json_key = jsondecode(var.use_existing_service_account ? (
+    base64decode(var.service_account_private_key)
+    ) : (
+    base64decode(module.lacework_gke_svc_account.private_key)
+  ))
+
+  log_filter = "protoPayload.@type=\"type.googleapis.com/google.cloud.audit.AuditLog\" AND protoPayload.serviceName = \"k8s.io\""
```

Comment:
Did we figure out if we need to add `container.googleapis.com` as well?

Reply:
Unfortunately no. I had asked about this on the original ticket. I guess, worst case scenario we can do a follow up PR to add it in. I don't think there is much hurry as this is still in quite early stages.